install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function install {
+  printf "\e[92m[~] Name your project: \e[0m\n"
+
+  read name
+
+  printf "\e[33m[~] Downloading awesomeness...\e[0m\n"
+
+  git clone https://github.com/mostjs/package-starter $name \
```

Comment:
i originally thought of doing something like `${name// /-}` to replace any spaces with dashes